.github/workflows/lint-eslint-core-ts.yml

```diff
@@ -32,6 +32,5 @@ jobs:
     - uses: AriPerkkio/eslint-remote-tester-run-action@v1
```

Comment:
Does Github Actions resolve these versions correctly, as in how semantic versioning suggests? Here is example run where workflow still loads old `v1` instead of `v1.1`:
https://github.com/AriPerkkio/eslint-remote-tester/runs/2368148819?check_suite_focus=true

Reply:
And here is one where `.yml` is referencing strictly to `v1.1`: https://github.com/AriPerkkio/eslint-remote-tester/runs/2368157696?check_suite_focus=true

Reply:
So the tag has to be manually created after publishing a release. 😒  https://github.com/actions/toolkit/blob/master/docs/action-versioning.md#recommendations